code/game/objects/structures/grille.dm

```diff
@@ -17,6 +17,10 @@
 	var/broken_type = /obj/structure/grille/broken
 	rad_flags = RAD_PROTECT_CONTENTS | RAD_NO_CONTAMINATE
 
+/obj/structure/grille/Destroy()
+	. = ..()
+	update_cable_icons_on_turf(get_turf(src))
```

Comment:
You want this before ..() since it will be nullspace otherwise.

Reply:
right, of course